CHANGES.md

```diff
@@ -225,7 +334,7 @@ OpenSSL 3.2
 
  * `CMS_add0_cert()` and `CMS_add1_cert()` no more throw an error
    if a certificate to be added is already present.
- * `CMS_sign_ex()` and `CMS_sign()` now ignore any duplicate certificates
+   `CMS_sign_ex()` and `CMS_sign()` now ignore any duplicate certificates
```

Comment:
If this bullet is being merged into the previous one, should the paragraph be reflowed (so that the first word appears on the line above)?